store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9929,4 +9929,10 @@ TODO: delete them permanently from here
   <defaultCOSValue>30</defaultCOSValue>
   <desc>Default calendar resolution preference</desc>
 </attr>
+<attr id="5000" name="zimbraGlobalExternalStoreConfig" type="string" cardinality="multi" optionalIn="globalConfig" since="9.1.0">
```

Comment:
Also, last added attr id was 404, not you start with 5000, why so?

Reply:
The id is not sequential, please go through the file. Some where it starts from 3xxx and for some attributes it starts for 5xxxx.
I started totally new series for storage management attributes.

Reply:
Done !